### PR TITLE
[16.0][MIG] edi_state_oca: Migration to 16.0

### PR DESCRIPTION
**Pull request by QuocDuong1306**
Depends on:
- #5

### CODE REVIEW

**Comment by simahawk** [comment]
to fwd port https://github.com/OCA/edi-framework/pull/17

File: test-requirements.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+odoo_test_helper
```
Comment (simahawk) [inline test-requirements.txt]: PR merged, you can drop this

**Review by simahawk — COMMENTED** [review]
LG overall

File: test-requirements.txt (hunk 1 above)
Comment (QuocDuong1306) [reply test-requirements.txt]: Thank @simahawk , I updated

**Review by simahawk — APPROVED** [review]

**Pull request merged**